packages/graphback-db-migrate/src/utils/logger.ts

```diff
@@ -0,0 +1,19 @@
+import * as pino from 'pino'
```

Comment:
Logging might need to be in the core to get the same approach in entire ecosystem

Reply:
Yeah, logging **was** in `graphback` package but this was causing a circular dependency, as `graphback` imported `@graphback/db-migrate`, and vice-versa. I moved it as this was the only package using it anyway. Separate issue or should this be resolved here?

Reply:
Moving it to core is 30 seconds. Creating issue is 2 minutes - and it will not be done anyway. Pick one :)

Reply:
I can do it now. I just thought it was something we weren't wanting to do yet.